src/arch/arm/imx/iomuxc/iomuxc.h

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file
+ *
+ * @date Nov 28, 2018
+ * @author Anton Bondarev
+ */
+
+#ifndef SRC_ARCH_ARM_IMX_IOMUXC_H_
+#define SRC_ARCH_ARM_IMX_IOMUXC_H_
+
+#include <stdint.h>
+
```

Comment:
Please add `IOMUXC_GPRN` addresses (they are used in ldb).

Reply:
Done